src/app/archive/broadcasts/broadcasts_date.component.ts

```diff
@@ -38,17 +38,17 @@ export class BroadcastsDateComponent {
     this.dateWithTimeSub = paramsObservable
       .subscribe(params => this.dateWithTime = this.getDateWithTime(params));
     this.broadcastsSub = dateObservable
-      .distinctUntilChanged(null, date => date.getTime())
-      .do(_ => this.loading = true)
+      .distinctUntilChanged(null, (date: Date) => date.getTime())
```

Comment:
Unable to find documentation about this `Observable#distinctUntilChanged()` method signature.

Official doc says it only has one param: http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#instance-method-distinctUntilChanged